ProjectSubmit/app/views/people/show.html.erb

```diff
@@ -1,40 +1,24 @@
 <section class="margin"> 
-  <div class="content">
-    <div class="content_container">
-      <article class="rounded">
-        <p id="notice"><%= notice %></p>
 
-        <h1><%= @person.name %></h1>
-        <%= image_tag @person.photo.url(:medium), :class => "rounded" %>
-        <p><%= @person.person_type.try(:designation) %></p>
-        <p>< <%= @person.email%> ></p>
-        <p>
-        </p>
-        <p>
-        </p>
-        <% if @person.juried_presentations.any? %>
-        <p>Foi juri em <%=@person.juried_presentations.size %> defesas de projectos.</p>
-        <% end %>
-      </article>
-    </div>  
-
-    <% #isto era uma magia negra para juntar projectos + orientações, este pedaço devia ir para um método no controller
-    projects_all = (@person.projects + @person.supervised_projects).sort_by(&:date)
-    %>
-
-    <div class="content_container">
-    <article class="rounded">
-    <h2>Projectos associados a <%= @person.name %></h2>
-      <% if projects_all.empty? %>
-        <p>Esta pessoa não participou em nenhum projecto.</p>
-      <% end %>
-    </article>
+<div class="content mt-1">
+    <div class="jumbotron px-1 py-1 rounded">
+        <div class="alert alert-success" role="alert">
+            <h1 class="text-xs-center">Visualizar Pessoas</h1>
+        </div>
+        <div id="notice"><%= notice %></div>
+        <table class="table table-hover table-sm">
```

Comment:
Usar tables para tudo caiu em desuso graças aos divs, agora usa-se se fizer sentido / for necessário. Isto depois é para a malta do design por direito ( @nelsonmpg @Pacn91 )